data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -876,7 +876,7 @@
             y=17
             side=1
             [not]
-                type=Dust Devil
+                formula=flying
```

Comment:
This is the sword in the pool, to fix #8188 the change should be on line 953 instead.

Reply:
Ugh, thanks for catching that. I tested in a copy. Need to figure out how to run the game from the checkout on macos at some point...